frontend/src/components/StockNews/StockNews.jsx

```diff
@@ -1,105 +1,97 @@
-        const selectedStock = stocks.find((stock) => stock.ticker === ticker);
-        setCurrentStock(selectedStock);
-    };
-
     return (
         <div className="stock-news">
             <h1>Related News</h1>
-            <select
-                className="stock-options"
-                value={currentStock?.ticker || ""}
-                onChange={handleOptions}
+            <DropdownButton
+                title={currentStock === null ? "Select a Stock" : currentStock}
+                className="mt-2 mb-3"
             >
-                <option value="" disabled defaultValue>
-                    Select a Stock You Follow
-                </option>
-                {stocks?.map((stock) => (
-                    <option key={stock.ticker} value={stock.ticker}>
-                        {stock.ticker}
-                    </option>
-                ))}
-            </select>
-            <div className="news">
-                {stockNews?.map((article) => (
-                    <div className="article-margin" key={article.url}>
-                        <a
-                            href={article.url}
-                            target="_blank"
-                            rel="noopener noreferrer"
-                            className="article-link"
+                {stocks?.map((stock) => {
+                    return (
+                        <Dropdown.Item
+                            key={stock.ticker}
+                            onClick={() => setCurrentStock(stock.ticker)}
                         >
-                            <div className="article">
-                                <div className="article-logo">
-                                    <img
-                                        src={currentStock.logo}
-                                        alt={`This is a logo of ${currentStock.name}.`}
-                                    />
-                                </div>
-                                <div className="article-content">
-                                    <h2>{article.title}</h2>
-                                    <p>{article.content}</p>
-                                    <p>
-                                        {formatDate(article.publishedAt)} •{" "}
-                                        {article.author}
-                                    </p>
-                                </div>
-                            </div>
-                        </a>
-                    </div>
-                ))}
-            </div>
+                            {stock.ticker}
+                        </Dropdown.Item>
+                    );
+                })}
+            </DropdownButton>
+            {stockNews?.slice(0, articlesToShow).map((article) => {
+                return (
+                    <Card
+                        key={article.id}
+                        className="mb-5 cursor-pointer"
+                        onClick={() => window.open(article.url, "_blank")}
```

Comment:
In the PR screenshot, it wasn't clear the article titles were hyperlinks. When you hover over, does it change the mouse to a pointer? Also clicking anywhere on the card should open the article, right?

Reply:
Correct